spec/features/admin_claim_with_missing_payroll_gender_spec.rb

```diff
@@ -22,7 +22,7 @@
 
     click_on "Save and continue"
 
-    expect(page).to have_content("You must select a gender that will be passed to HMRC")
+    expect(page).to have_content("Select male, female, or I don’t know")
```

Comment:
Please see the note above relating to payroll gender

Reply:
I have updated this to v10 now